api/v1beta1/hostedcluster_types.go

```diff
@@ -781,6 +781,12 @@ type PowerVSPlatformSpec struct {
 	//
 	// +immutable
 	StorageOperatorCloudCreds corev1.LocalObjectReference `json:"storageOperatorCloudCreds"`
+
```

Comment:
Please let's move anything powervs specific under `.PowerVSPlatformSpec`, there's already refs for other creds there.

Reply:
Newly added `ImageRegistryOperatorCloudCreds` is under `PowerVSPlatformSpec` only.